{"name": "test_sdbaseline_03", "status": "passed", "description": "03. failure case: call sdbaseline with invalid image stack, an exception raise.", "start": 1758038160331, "stop": 1758038160356, "uuid": "88e523ce-24bb-4599-9ffe-cc586c9e36c3", "historyId": "7378780edf33888a037215d6075c0b27", "testCaseId": "7378780edf33888a037215d6075c0b27", "fullName": "casatasks.test_task_imbaseline.TestSdbaseline#test_sdbaseline_03", "labels": [{"name": "parentSuite", "value": "casatasks"}, {"name": "suite", "value": "test_task_imbaseline"}, {"name": "subSuite", "value": "TestSdbaseline"}, {"name": "host", "value": "casa-rockylinux8-amd64-perf07"}, {"name": "thread", "value": "597558-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "casatasks.test_task_imbaseline"}], "titlePath": ["casatasks", "test_task_imbaseline.py", "TestSdbaseline"]}